Dockerfile

```diff
@@ -48,69 +47,72 @@ RUN apt-get -y install \
     latex-cjk-chinese-arphic-gbsn00lp latex-cjk-chinese-arphic-gkai00mp \
     fonts-symbola
 
-# Install 'rsvg-convert' for SVG -> PDF conversion
+# plantuml: is to support sphinxcontrib-plantuml
+# https://pypi.org/project/sphinxcontrib-plantuml/
+#
+# imagemagick: is to support sphinx.ext.imgconverter
+# http://www.sphinx-doc.org/en/master/usage/extensions/imgconverter.html
+#
+# rsvg-convert: is for SVG -> PDF conversion
 # using Sphinx extension sphinxcontrib.rsvgconverter, see
 # https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter
-RUN apt-get -y install librsvg2-bin
-
-# Install plantuml
-RUN apt-get -y install plantuml
+#
+# swig: is required for different purposes
+# https://github.com/rtfd/readthedocs-docker-images/issues/15
+RUN apt-get -y install plantuml imagemagick librsvg2-bin swig
 
 # Install Python tools/libs
 RUN apt-get -y install python-pip && pip install -U virtualenv auxlib
 
+# Install jsdoc and typedoc
+RUN apt-get -y install nodejs npm && npm install --global jsdoc typedoc
+
 # UID and GID from readthedocs/user
 RUN groupadd --gid 205 docs
 RUN useradd -m --uid 1005 --gid 205 docs
 
-# Install jsdoc and typedoc
-RUN apt-get -y install nodejs npm && npm install --global jsdoc typedoc
-
 USER docs
 WORKDIR /home/docs
 
 # Install Conda
-RUN curl -O https://repo.continuum.io/miniconda/Miniconda2-${CONDA_VERSION}-Linux-x86_64.sh
-RUN bash Miniconda2-${CONDA_VERSION}-Linux-x86_64.sh -b -p /home/docs/.conda/
+RUN curl -O https://repo.continuum.io/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh
+RUN bash Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh -b -p /home/docs/.conda/
 ENV PATH $PATH:/home/docs/.conda/bin
+RUN rm -f Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh
 
 # Install pyenv
-RUN git clone --depth 1 https://github.com/yyuu/pyenv.git ~docs/.pyenv
+RUN wget https://github.com/pyenv/pyenv/archive/master.zip
+RUN unzip master.zip && \
+    rm -f master.zip && \
+    mv pyenv-master ~docs/.pyenv
```

Comment:
Why moving to a non-release?

Reply:
We were cloning and using `master` before. Now, instead of clonning, I'm downloading the .zip file to avoid all the git metadata that's useless.